src/compiler/output-targets/dist-hydrate-script/generate-hydrate-app.ts

```diff
@@ -76,12 +75,9 @@ export const generateHydrateApp = async (
 const generateHydrateFactory = async (config: d.ValidatedConfig, compilerCtx: d.CompilerCtx, buildCtx: d.BuildCtx) => {
   if (!buildCtx.hasError) {
     try {
-      const cmps = buildCtx.components;
-      const build = getHydrateBuildConditionals(config, cmps);
-
       const appFactoryEntryCode = await generateHydrateFactoryEntry(buildCtx);
 
-      const rollupFactoryBuild = await bundleHydrateFactory(config, compilerCtx, buildCtx, build, appFactoryEntryCode);
```

Comment:
Here we remove this arg that corresponds to [the unused param](https://github.com/ionic-team/stencil/pull/4623/files#r1274924140). `build`, declared above, is no longer used. We can then delete it, `getHydrateBuildConditionals`, `cmps`, and the imports related to these things